convert_to_hdf5/src/convert_to_hdf5.py

```diff
@@ -188,6 +190,56 @@ def load_hdf_files(record_names, out_dir, data_info_mapping, meta_data, topic_in
                 rospy.loginfo('\t\tNo meta info')
     return hdf5_files
 
+def match_depth(hdf5_files, data_info_mapping):
+    upper_color_match_topic = 'vid/color/data/upper/matched_depth_index'
+    lower_color_match_topic = 'vid/color/data/lower/matched_depth_index'
+
+    timestamps = {}
+    dataset_upper = None
+    dataset_lower = None
+
+    for hdf5_database in hdf5_files:
+        if upper_color_match_topic not in hdf5_database :
+            dataset_upper = hdf5_database.create_dataset(upper_color_match_topic, (INITIAL_SIZE,),
```

Comment:
you know the size here, right? This will have the same length as the color sequence.

Reply:
Missed this

Reply:
I don't know if it is better this way for consistency?